Cost task
---
string error_code
bool is_taken
